scrapy/downloadermiddlewares/httpproxy.py

```diff
@@ -1,14 +1,14 @@
 import base64
-from six.moves.urllib.request import getproxies, proxy_bypass
-from six.moves.urllib.parse import unquote
```

Comment:
We usually put six.moves imports in the same group as stdlib import, because in a way they are stdlib imports, just with locations compatible with Python 2 and Python 3. When Scrapy supported only Python 2, these imports were in a group of stdlib imports, when Scrapy will support only Python 3, these imports will again be just stdlib imports. This should be rather consistent in other Scrapy files as well; I'd prefer not to change it.

Reply:
I agree. I will just put the two imports from six.moves.urllib.parse together.